addons/fluxcd/readme.md

```diff
@@ -16,3 +16,38 @@ Enable fluxcd addon to use these X-definitions
 git repo / helm repo / S3 compatible bucket.
 
 - [kustomize](https://kubevela.io/docs/end-user/components/kustomize) helps to deploy a kustomize style artifact.
+
+## Note
+- In this Fluxcd Addon, there are five controllers to be installed by default
+
+```shell
+USED BY CONTROLLER              DEFINITION NAME                         DEFINITION TYPE           DEFINITION DESCRIPTION
+helm-controller                config-helm-repository                  ComponentDefinition       Config information to authenticate helm chart repository
+helm-controller                helm                                    ComponentDefinition       helm release is a group of K8s resources from either git
+                                                                                                 repository or helm repo
+kustomize-controller           kustomize                               ComponentDefinition       kustomize can fetching, building, updating and applying
+                                                                                                 Kustomize manifests from git repo.
+kustomize-controller           kustomize-json-patch                    TraitDefinition           A list of JSON6902 patch to selected target
+kustomize-controller           kustomize-patch                         TraitDefinition           A list of StrategicMerge or JSON6902 patch to selected
+                                                                                                 target
+kustomize-controller           kustomize-strategy-merge                TraitDefinition           A list of strategic merge to kustomize config
+```
```

Comment:
You can use the following syntax to show table in markdown:

```
| Syntax      | Description |
| ----------- | ----------- |
| Header      | Title       |
| Paragraph   | Text        |
```

refer to https://www.markdownguide.org/extended-syntax/#tables

Reply:
OK, Thanks, ^_^